WordPress/Classes/ViewRelated/Me/App Settings/Boolean User Defaults/BooleanUserDefaultsDebugView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct BooleanUserDefaultsDebugView: View {
+    @StateObject private var viewModel = BooleanUserDefaultsDebugViewModel()
+
+    var body: some View {
+        List {
+            ForEach(viewModel.userDefaultsSections.keys.sorted(), id: \.self) { sectionKey in
+                let userDefaultsSection = viewModel.userDefaultsSections[sectionKey] ?? BooleanUserDefaultEntries()
+
+                Section(header: Text(sectionKey)
+                    .font(.caption)) {
+                        ForEach(userDefaultsSection.keys.sorted(), id: \.self) { userDefaultKey in
+                            let isOn = Binding<Bool>(
+                                get: { userDefaultsSection[userDefaultKey]?.value ?? false },
+                                set: { newValue in viewModel.updateUserDefault(newValue, forSection: sectionKey, forUserDefault: userDefaultKey) }
+                            )
+                            Toggle(userDefaultsSection[userDefaultKey]!.title, isOn: isOn)
```

Comment:
I would rather we don't force unwrap a Dictionary. Can we simply pass in an optional in `Toggle`? If not, can we do it the same way as you did above?
`userDefaultsSection[userDefaultKey]?.title ?? ""`

Reply:
Good catch, Alp. Thanks! Already handled it.